app/app/management/commands/sync_orgs_repos.py

```diff
@@ -0,0 +1,189 @@
+                            (handle, access_token)
+                        )
+
+                        if 'message' in org_repos:
+                            print(org_repos['message'])
+                            continue
+
+                        for repo in org_repos:
+                            db_repo = Repo.objects.get_or_create(name=repo['name'])[0]
+                            db_org.repos.add(db_repo)
+                            print(f'Syncing Repo: {db_repo.name}')
+                            repo_collabs = get_repo(
+                                repo['full_name'],
+                                '/collaborators',
+                                (handle, access_token)
+                            )
+                            if 'message' in repo_collabs:
+                                print(repo_collabs['message'])
+                                continue
+
+                            for collaborator in repo_collabs:
+
+                                if collaborator['permissions']['admin']:
+                                    permission = "admin"
+                                elif collaborator['permissions']['push']:
+                                    permission = "write"
+                                elif collaborator['permissions']['pull']:
+                                    permission = "pull"
+                                else:
+                                    permission = "none"
+
+                                db_group = Group.objects.get_or_create(
+                                    name=f'{db_org.name}-repo-{repo["name"]}-{permission}')[0]
+                                db_org.groups.add(db_group)
+
+                                try:
+                                    member_user_profile = Profile.objects.get(handle=collaborator['login'],
+                                                                              user__is_active=True)
+                                    member_user_profile.user.groups.add(db_group)
+                                    member_user_profile.repos.add(db_repo)
+                                    if collaborator['login'] not in members_to_sync or \
+                                        collaborator['login'] not in lsynced:
+                                        members_to_sync.append(collaborator['login'])
+                                except Exception as e:
+                                    continue
+
+                        for x in members_to_sync:
+                            try:
+                                lsynced = lsynced + recursive_sync(lsynced, x)
+                            except Exception as e:
+                                # print(f'An exception happened in the Members sync Loop: handle: {handle} {e}')
+                                continue
+                    except Exception as e:
+                        # print(f'An exception happened in the Organization Loop: handle {handle} {e}')
```

Comment:
^ do we wanna remove the commented out statements

Reply:
I realized they are redundant for production but helpful for development, either they get changed in terms of their messaging or removed, they will appear a lot when syncing and getting a user that is outside of the current dataset of users inside gitcoin. 

we only synchronize profiles for users who are currently a gitcoin user.